### DISCUSSION TITLE
onAllFuturesCompleted for MultipleFutureViewModel

### DISCUSSION THREAD

**Discussion by eivihnd** [FilledStacks/stacked#947] [Ideas]
I have a case where it would be beneficial to be able to call a function when all futures have completed in a MultipleFutureViewModel. Would it make sense to add a call to `onAllFuturesCompleted` that we can override in our viewModels?

Or are there better ways (already in place?) to achieve the same?

I am happy to submit a PR if interested.

**Comment by FilledStacks** [comment]
Hey @eivihnd there is no callback but it should be easy to add into the package. 

We have a `_futureCompleter` that we complete when it's ready to complete. After that you can simply fire the callback.


It would be awesome if you can submit a PR.

**Comment by eivihnd** [comment]
Fixed in version 3.3.0
